public/app/features/dashboard/components/PanelEditor/getPanelFrameOptions.tsx

```diff
@@ -71,6 +72,7 @@ export function getPanelFrameCategory(props: OptionPaneRenderProps): OptionsPane
         addon: config.featureToggles.dashgpt && (
           <GenAIPanelDescriptionButton onGenerate={setPanelDescription} panel={panel} />
         ),
+        panelType: panel.type,
```

Comment:
I'd try to avoid adding a new prop to the Options definition as much as possible. It is already complex interface. If the problem is only thins one, maybe we can manage the state with `onBlur` and the value targeting to the input value through ref? Just brainstorming here